README.md

```diff
@@ -22,7 +22,18 @@ kubectl rook-ceph ceph status
 
 ## Commands
 
-TBD
+1. We support running Ceph command with `rook-ceph` [krew]([Krew](https://github.com/kubernetes-sigs/krew)) plugin
+   1. Example:
+
+       1.
+        ```console
+        # we could also provide optional flag namespace with `-n` or `--namespace` after `kubect rook-ceph` and before main commands.
+        # example: `kubectl rook-ceph` as prefix with `ceph` command
+        kubectl rook-ceph <-n/--namespace=<namespace> ceph status
+        ```
+        2. `kubectl rook-ceph ceph --status --format=json-pretty`
+
+2. Restart rook-ceph operator `kubectl rook-ceph operator restart`.
```

Comment:
I think it might be easier for users to see at a glance if this is written more like a `man` page where the command is given first. 

Something like this explains overall usage to users first, then calls out each supported command, which is pretty standard for how commands are documented for users. IMO, it's fine to put examples either first or last. I have seen both.

```md
### Overall usage
`kubectl rook-ceph <root-args> <command> <command-args>`

### Root args
- `--namespace` | `-n`: the Kubernetes namespace in which the CephCluster resides (default: rook-ceph)

### Commands
- `ceph`: Run a Ceph CLI command. Supports any arguments the `ceph` command supports. See [Ceph docs](link) for more.
- `operator`: run sub-commands against the Rook-Ceph operator
  -  `restart`: restart the Rook-Ceph operator

### Examples
- Run a ceph command: `kubectl rook-ceph ceph --status --format=json-pretty`
- Run a command against a CephCluster in a different namespace: `kubectl rook-ceph --namespace my-cluster ceph --status` 
- Restart the operator: `kubectll rook-ceph operator restart`
```

There are different strategies for keeping documentation up-to-date with a CLI tool. It seems beneficial to note for users what commands the tool supports so they can hopefully decide if it's worth using. Maybe we should just copy the usage text here, either manually or with an automated build script.

Reply:
That approach looks good to me too. To keep it simple, maybe we just keep it like that with the most common commands listed here, then point out to use --help for more options.